Prime Checker/prime checker c.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+
+int prime(int n)
+{
+	int i;
+  	if (n == 1) 
+	{
+    return 0;

```

Comment:
Just to point one example of miss indentation we can point this line, this return should be inside the if block indentation

Reply:
I have made the changes in the indentation. I don't know why in one particular line it is showing an extra tab but there is no such space when the file is opened.